pkg/controllerutils/finalizers.go

```diff
@@ -0,0 +1,45 @@
+package controllerutils
```

Comment:
Don't need a separate package for this, move this to the `controllers` package

Reply:
Updated.